test/unit/region.spec.js

```diff
@@ -122,18 +122,17 @@ describe('region', function() {
         onSwapOut: function() {}
       });
 
-      this.MyView = Backbone.View.extend({
+      this.MyView = Marionette.View.extend({
```

Comment:
Ah yeah interesting.  `onBeforeRender` would not have worked on a vanilla View.  Seems like the vanilla view test would be much less involved as well.  Would it be easier just to make a simpler test case well documented for testing the vanilla view, or is that just too much extra stuff?